src/main/webapp/scripts/user-script.js

```diff
@@ -54,19 +61,77 @@ function addOrgUnits2Data(ous){
         }
         break;
     }
-    
+}
+
+function addOrgUnits2DataSearch(ous){
```

Comment:
maybe rename to addOrgUnitsToDataSearch?

Reply:
changes in new commit